test/integration/generator/content/schedule/part3/index.test.js

```diff
@@ -0,0 +1,67 @@
+jest.mock('../../../../../../app/generator/content/schedule/part3/get-table')
+const { getTable } = require('../../../../../../app/generator/content/schedule/part3/get-table')
+
+const part3 = require('../../../../../../app/generator/content/schedule/part3')
+const { topUpSchedule, reductionSchedule, reductionZeroSchedule } = require('../../../../../mocks/mock-schedule')
+
+describe('schedule part 3 integration with getSummary components', () => {
+  afterEach(() => {
+    jest.clearAllMocks()
+  })
+
+  test('Ensure getTable is called once', () => {
```

Comment:
Remmove test.

This is a unit test not integration.
This is duplicate from here: https://github.com/DEFRA/ffc-pay-statement-generator/pull/51/files#diff-ba0b49ed636c32320433e3ce38d8eafa14b83020b08918d89a4803b366bfbc4dR16
For each scenario not just top-up